### PR TITLE
feat(Button): new component

### PR DESCRIPTION
**Pull request by GaroGabrielyan**
### Chromatic Build Summary 
**Build URL:** https://www.chromatic.com/build?appId=66daee7859a843dd87bfb3d7&number=139 
**Storybook URL:** https://66daee7859a843dd87bfb3d7-dovlprusfq.chromatic.com/

### CODE REVIEW

File: src/lib/atoms/Button/Button.test.tsx [hunk 1]
```diff
@@ -1,50 +1,70 @@
-import React from 'react';
+import React, { MouseEvent } from 'react';
 import { ReactWrapper, mount } from 'enzyme';
 
 // Components
 import Button, { IButtonProps } from './index';
+import GeneUIProvider from '../../providers/GeneUIProvider';
 
-describe('Button', () => {
+describe('Button ', () => {
```
Comment (GaroGabrielyan) [inline src/lib/atoms/Button/Button.test.tsx:8]: I can't find the `isIconAfter` prop test

Comment (NarekArshakyan) [reply src/lib/atoms/Button/Button.test.tsx:8]: Test with icons i  will add after finish Icon component.

File: src/lib/atoms/Button/Button.stories.tsx [hunk 2]
```diff
@@ -5,68 +5,36 @@ import { Meta } from '@storybook/react';
...
 // Components
-import ButtonComponent, { IButtonProps } from './index';
+import Button, { IButtonProps } from './index';
 
-const meta: Meta<typeof ButtonComponent> = {
+const meta: Meta<typeof Button> = {
     title: 'Atoms/Button',
-    component: ButtonComponent,
+    component: Button,
     argTypes: {
```
[the remark this answers is not in the document]
Comment (NarekArshakyan) [reply src/lib/atoms/Button/Button.stories.tsx:13]: I'll add it when the loader class is ready

File: src/lib/atoms/Button/Button.stories.tsx [hunk 3]
```diff
@@ -5,68 +5,36 @@ import { Meta } from '@storybook/react';
...
-        flexibility: args({ control: 'select', defaultValue: 'default', ...propCategory.appearance }),
-        itemsDirection: args({ control: 'select', defaultValue: 'start', ...propCategory.appearance })
+        appearance: args({ control: 'select', ...propCategory.appearance }),
+        text: args({ control: 'text', ...propCategory.content }),
+        type: args({ control: 'select', ...propCategory.appearance }),
+        fullWidth: args({ control: 'boolean', ...propCategory.appearance }),
+        disabled: args({ control: 'boolean', ...propCategory.functionality }),
+        Icon: args({ control: 'false', ...propCategory.content }),
+        name: args({ control: 'false', ...propCategory.functionality }),
+        onClick: args({ control: 'false', ...propCategory.functionality }),
```
[the remark this answers is not in the document]
Comment (NarekArshakyan) [reply src/lib/atoms/Button/Button.stories.tsx:22]: I think functionality is more suitable for it.

File: src/lib/atoms/Button/Button.stories.tsx [hunk 4]
```diff
@@ -5,68 +5,36 @@ import { Meta } from '@storybook/react';
...
+        isIconAfter: false
     }
 };
 
 export default meta;
 
-const Template: FC<IButtonProps> = ({ children, ...args }) => {
-    return <ButtonComponent {...args}>{children}</ButtonComponent>;
-};
+const Template: FC<IButtonProps> = (args) => <Button {...args} />;
```
[the remark this answers is not in the document]
Comment (NarekArshakyan) [reply src/lib/atoms/Button/Button.stories.tsx:36]: There are many options here, no need, everything can be controlled through props

**Review by hamikhambardzumyan — APPROVED** [review]

**Pull request merged**